cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -22,6 +22,11 @@ service PackagesService {
       get: "/core/packages/v1alpha1/packagerepositories"
     };
   }
+  rpc GetPackageMeta(kubeappsapis.core.packages.v1alpha1.GetPackageMetaRequest) returns (kubeappsapis.core.packages.v1alpha1.GetPackageMetaResponse) {
```

Comment:
Can we just use `GetInstalledPackageDetails`? Fine if not, we'll be updating it soon.

Reply:
one thing at a time :-). It will probably be in my next PR